src/saveload/ai_sl.cpp

```diff
@@ -49,12 +55,21 @@ static void SaveReal_AIPL(int *index_ptr)
 		_ai_saveload_version = -1;
 	}
 
-	_ai_saveload_is_random = config->IsRandom();
```

Comment:
Doesn't `_ai_saveload_is_random` now remain unset? As you do read it a bit later on as if it is always set. Guess you need to force it to false if the new description exists?

Reply:
It's not saved, and it's set to false before loading. But the variable will be read from old saves.